src/scancode/cli.py

```diff
@@ -321,6 +323,10 @@ def scancode(ctx,
     Error and progress is printed to stderr.
     """
 
+    pm = pluggy.PluginManager('scan_proper')
```

Comment:
I've used `pre_scan` `post_scan` markers, for the respective scan levels. Pertaining to those `scan_proper` made sense, I'd be happy to change it if you like. Here are some of my suggestions `proper_scan` `in_scan` or `scan_cli`.


_Sent from my Xiaomi MI 4W using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_